packages/app/src/app/components/TeamAvatar/TeamAvatar.tsx

```diff
@@ -15,29 +15,51 @@ export const backgrounds = [
 
 interface TeamAvatarProps {
   name: string;
-  size?: 'small' | 'big';
+  size?: 'small' | 'big' | 'bigger';
   className?: string;
+  avatar?: string;
+  style?: any;
```

Comment:
Nice helper function for this:

```suggestion
  style?: React.CSSProperties;
```